static/app/components/events/contexts/contextCard.spec.tsx

```diff
@@ -0,0 +1,118 @@
+import startCase from 'lodash/startCase';
+import {EventFixture} from 'sentry-fixture/event';
+import {GroupFixture} from 'sentry-fixture/group';
+
+import {initializeOrg} from 'sentry-test/initializeOrg';
+import {render, screen} from 'sentry-test/reactTestingLibrary';
+
+import ContextCard from 'sentry/components/events/contexts/contextCard';
+
+describe('ContextCard', function () {
+  it('renders the card with formatted context data', function () {
+    const event = EventFixture();
+    const group = GroupFixture();
+    const {project} = initializeOrg();
+    const alias = 'Things in my Vicinity';
+    const simpleContext = {
+      snack: 'peanut',
+      dinner: 'rice',
+      friend: 'noelle',
+    };
+    const structuredContext = {
+      'my dogs': ['cocoa', 'butter'],
+      book: {
+        title: 'This Is How You Lose the Time War',
+        pages: 208,
+        published: new Date('2018-07-21'),
```

Comment:
would this be a date or an iso string from the api

Reply:
Good catch, yeah the API returns ISO strings -- I'll update this.